src/scripts/test_cli.py

```diff
@@ -975,10 +969,7 @@ def cli_tls_http_server_tests(tmp_dir):
     if resp.status != 405:
         logging.error('Unexpected response status %d', resp.status)
 
-    if sys.version_info.major >= 3:
-        rc = tls_server.wait(5) # pylint: disable=too-many-function-args
-    else:
-        rc = tls_server.wait()
+    rc = tls_server.wait(5) # pylint: disable=too-many-function-args
```

Comment:
I think this pylint annotation can be removed now, it was getting confused between 2.x and 3.x APIs

Reply:
LGTM (and to the linter): https://github.com/randombit/botan/actions/runs/3068176208/jobs/4955320144